lib/logger.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Abtracted logging properties and methods.
+ *
+ * If for any reason, they need a different logging tool or need to be extended,
+ * the changes can be made here. 
+ */
+var util = require('util');
+var logger = require('winston');
+
+exports.logger = logger;
+
+exports.isTest = function () {
+  'use strict';
+
+  var isTest = false;
+
+  for (var i = 0; i < process.argv.length; i++) {
+    if (process.argv[i].substr(-5) === 'mocha') {
+      isTest = true;
+      break;
+    }
+  }
+
+  return isTest;
+};
+
+exports.i = function (obj, showHidden, depth) {
```

Comment:
@scottnath Sorry. I copied this from my personal code collection without modification. I updated it so other know what it's doing.